examples/backend/src/ExampleAPI.hs

```diff
@@ -57,11 +89,34 @@ albert = User {name = "Albert", age = 18}
 
 server :: Server UserAPI
 server =
-  return users
-    :<|> return albert
+  simpleRequests
     :<|> return
```

Comment:
You should also factor this one, for consistency with the other handlers (and clarity too).

Reply:
done